.env.example

```diff
@@ -1,5 +1,5 @@
-CLOUDINARY_URL=cloudinary://your_api_key:your_api_secret@your_cloud_name
-MONGO_URI=mongodb+srv://your_username:your_password@your_cluster_url/your_database?retryWrites=true&w=majority&appName=YourAppName
+CLOUDINARY_URL=""
+MONGO_URI=""
 SENTRY_DSN=
```

Comment:
this was example don't think this should be removed

Reply:
oh i removed my creds will revert